app/services/anonymized_data/district_data.rb

```diff
@@ -0,0 +1,40 @@
+class AnonymizedData::DistrictData
+  attr_reader :facilities
+
+  def initialize(district)
+    @facilities = district.facilities
+  end
+
+  def raw_data
```

Comment:
We could probably name this method better than `raw_data`? WDYT?

Reply:
@kitallis Yes. I cannot think of a very apt name though. Any suggestions? I thought of `anonymized_data`, but then we'd have to probably change the `anonymize` method's name as well then.

Reply:
Okay, renamed `raw_data` to `fetch` as per discussion.